db/models.py

```diff
@@ -1 +1,48 @@
 from django.db import models
+
+
+RACES_NAME = [
+    ("Elf", "Elf"),
+    ("Human", "Human"),
+    ("Dwarf", "Dwarf"),
+    ("Orc", "Orc")
+]
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True, choices=RACES_NAME)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(
+        Race,
+        related_name="skill_set",
```

Comment:
```suggestion
        related_name="skills",
```

Reply:
Tests break if it will be 'skills', I was try it first